okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -96,27 +98,58 @@
     assertEquals(expectedEvents, listener.recordedEventTypes());
   }
 
-  @Test public void successfulHttpsCallEventSequence() throws IOException {
-    enableTlsWithTunnel(false);
-    server.enqueue(new MockResponse()
-        .setBody("abc"));
-
+  private void assertSuccessfulEventOrder() throws IOException {
     Call call = client.newCall(new Request.Builder()
-        .url(server.url("/"))
-        .build());
+            .url(server.url("/"))
+            .build());
```

Comment:
@swankjesse my bad, fresh Intellij install.  Is there an automated way to reformat?